lib/perl/Genome/Model/Tools/Capture/FormatSnvs.t

```diff
@@ -4,10 +4,56 @@ use strict;
 use warnings;
 
 use above 'Genome';
-use Test::More tests => 1;
+use Test::More;
 
-# This test was auto-generated because './Model/Tools/Capture/FormatSnvs.pm'
-# had no '.t' file beside it.  Please remove this test if you believe it was
-# created unnecessarily.  This is a bare minimum test that just compiles Perl
-# and the UR class.
 use_ok('Genome::Model::Tools::Capture::FormatSnvs');
+
+BEGIN {
+  $ENV{UR_DBI_NO_COMMIT} = 1;
+  $ENV{UR_USE_DUMMY_AUTOGENERATED_IDS} = 1;
+}
+
+my $expected_output_dir = $ENV{"GENOME_TEST_INPUTS"} . "Genome-Model-Tools-Capture-FormatSnvs/2015-02-12/";
+ok(-e $expected_output_dir, "Found test dir: $expected_output_dir") or die;
+
+my $variants_file = File::Spec->join(
+  $expected_output_dir,
+  "varscan.snps",
+);
+ok(-e $variants_file, "Found variant file: $variants_file") or die;
+
+#my $temp_dir = Genome::Sys->create_temp_directory();
+my $temp_dir = ".";
+ok($temp_dir, "created temp directory: $temp_dir") or die;
+
+my $formatter = Genome::Model::Tools::Capture::FormatSnvs->create(
+  variants_file => $variants_file,
+  outdir => $temp_dir,
+);
+$formatter->execute();
+
+my $expected_outfile = File::Spec->join(
+  $expected_output_dir,
+  "varscan.snps.filtered"
+);
+ok(-e $expected_outfile, "Found expected outfile: $expected_outfile") or die;
+my $outfile = File::Spec->join(
+  $temp_dir,
+  "varscan.snps.filtered",
+);
+ok(-e $outfile, "Found outfile: $outfile") or die;
+
+#Perform a diff between the stored results and those generated by this test
+my @diff = `diff -r $outfile $expected_outfile`;
```

Comment:
`Genome::Utility::Test` also provides a test that might be handy here.
```perl
use Genome::Utility::Test qw(compare_ok); #as before but exporting compare_ok
compare_ok($expected_outfile, $outfile);
```
If there is a diff, the output is something like:
```
# First diff:
# --- /expected/file.out line 1
# +++ /actual/file.out line 1
# -[expected]
# +[actual]
not ok 2
```